CHANGELOG.md

```diff
@@ -157,7 +163,7 @@ for further information about branching and tagging conventions.
 - A feature that makes your tests execution fail if there’s any `console.log`,
   `console.warn` or `console.error` call
 - Account reset related stuff:
-  - `isAccountRoleReseted` state field to the KYC vuex module
+  - `isAccountRoleReset` state field to the KYC vuex module
```

Comment:
Restore `isAccountRoleReset`. We did it intentionally to avoid confusing with `reset` as a mode or current state or something

Reply:
Updated